.travis.yml

```diff
@@ -23,49 +22,55 @@ before_script:
      faketime
      geoip-database
      ldnsutils
-     boost1.55
+     libboost-all-dev
      libbotan1.10-dev
      libcdb-dev
-     libcdb-dev
      libcurl4-openssl-dev
+     libgeoip-dev
      libldap2-dev
-     liblua5.1-posix1
+     liblmdb-dev
+     liblmdb0
      libnet-dns-perl
      libopendbx1-dev
      libopendbx1-sqlite3
      libp11-kit-dev
      libtolua-dev
      libtool
+     libyaml-cpp-dev
+     libzmq3-dev
      links
+     lmdb-utils
+     lua-posix
      moreutils
      p11-kit
+     pandoc
      pkg-config
+     python-openssl
      python-virtualenv
      rpm
      ruby-json
      ruby-sqlite3
      ruby1.9.1
-     rubygems
      socat
      softhsm
      time
      unbound-host
      xmlto
- - sudo sh -c 'sed s/precise/trusty/g /etc/apt/sources.list > /etc/apt/sources.list.d/trusty.list'
- - sudo apt-get update --quiet --quiet
- - sudo apt-get install --quiet --quiet libyaml-cpp-dev libzmq3-dev pandoc libgeoip-dev
- - sudo pip install pandocfilters==1.2.3 mkdocs==0.14 linkchecker==9.3 click==5.1
- - sudo update-alternatives --set ruby /usr/bin/ruby1.9.1
+ - virtualenv $HOME/.venv
+ - source $HOME/.venv/bin/activate
+ - pip install pyopenssl ndg-httpsclient pyasn1
```

Comment:
just to be sure, `pip` was giving me a bad time installing pandocfilters because of requests complaing about ssl.